### DISCUSSION TITLE
CPU overload and proper networking

### DISCUSSION THREAD

**Discussion by Nick3Nick** [apohl79/audiogridder#1116] [Help] [answered]
Hello all,

I've just started using AG today. Here's where I'm at:

1. I have my main laptop running pro tools 2022.10 | AMD Ryzen 7 5800H, 8 cores | 16gb ram. Connected directly (via ethernet) to server laptop of similar spec (i7, 16gb of ram) just a few years older. I can connect and use the system while mixing. Is this optimal way to set up the network?
2. I've come to a point where I only get a 9173 error in pro tools and can no longer playback. Stats show healthy feedback that does not reflect using up all available CPU. I'm using 5 instances of the AG plugin running Acustica plugins.

I've mixed sessions of similar size without AG and not had this many errors.

Any insight is appreciated.

Thank you! Nick

**Comment by oldtos** [comment]
Make sure it is Gigabit ethernet....  check buffer sizes in both PT and Audiogridder ...

**Reply by Nick3Nick** [reply]
Good idea. Switching out ethernet cable now. What's a good starting point for buffer sizes? Or is it a trial and error thing?

Thank you!

**Comment by Nick3Nick** [comment, marked as answer]
Switched the cable to a crossover ethernet cable (Allows two-way transfer. Best for connecting peer-to-peer like PC to PC). Also set each PC to automatically detect and set up the ip addresses.

These two changes made all the difference and now have full stability. Successfully had 20+ instances of AG plugin running CPU heavy plugins simultaneously. And am confident I can add many more.
